utils/build/docker/python/uwsgi-poc.base.Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN python --version && curl --version
 
```

Comment:
Can you update the version number of this base image ? look for uwsgi-poc.base in the code